Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM manjarolinux/base:latest
+
+ARG VERSION=1.29.0
+
+# This will run as root
```

Comment:
This comment confuses me a bit since it doesn't appear as if this is being run as root nor is the other command? But I could be missing something?

Reply:
Commands run during Dockerfile builds are run as root my default. Updated the comment to be more explicit...